src/vm/moar/runner/main.c

```diff
@@ -186,8 +186,10 @@ int wmain(int argc, wchar_t *wargv[])
     int flag;
     int new_argc     = 0;
 
+#ifdef HAVE_TELEMEH
```

Comment:
Have you considered moving these variable declarations down into the existing ifdef in line 262? Would save some clutter.

Reply:
Inside the `if (getenv("MVM_TELEMETRY_LOG")) {`? But then `interval_id` wouldn't be available at https://github.com/rakudo/rakudo/pull/3393/files/bebd268f6a4ccb8e5cdd96b2d74da015065f34d3..293a0b631fe93fbe2ab3858d3b7455e62a71baad#diff-61d548d860ffcb20f03c63354f4132c0L386

Reply:
No, just before the if:
```
#ifdef HAVE_TELEMEH
    unsigned int interval_id = 0;
￼    char telemeh_inited = 0;
    if (getenv("MVM_TELEMETRY_LOG") && telemeh_inited) {
```